web/common.inc.sh

```diff
@@ -94,7 +94,7 @@ function test-headless() {
     TEST_OPTS="--reporter mocha-teamcity-reporter"
   fi
   if [[ -n "$TEST_EXTENSIONS" ]]; then
-    TEST_OPTS=" --extension $TEST_EXTENSIONS"
+    TEST_OPTS="$TEST_OPTS --extension $TEST_EXTENSIONS"
```

Comment:
I happened to notice that Web's child build scripts weren't outputting their headless-testing results into TC-friendly format even when run by CI.  It was a simple fix, and it's kind of related anyway, so I threw the one-liner fix in here.